e2e/e2e-prow.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### This script is used by Kubernetes Test Infrastructure to run integration tests.
+### See documenation at https://github.com/kubernetes/test-infra/tree/master/prow
+
+set -ex
+
+# Include requisite scripts
+source ./include/util.sh
+
+TEST_ROOT_DIR=$(git rev-parse --show-toplevel)
+BRANCH="master"
+SPARK_REPO="https://github.com/apache/spark"
+SPARK_REPO_LOCAL_DIR="$TEST_ROOT_DIR/target/spark"
+
+## Install basic dependencies
+echo "deb http://http.debian.net/debian jessie-backports main" >> /etc/apt/sources.list
```

Comment:
I get nervous when scripts like this install packages on the node they're running on. Also, it makes this script debian-specific. Is there a reason to do it this way as opposed to have another script with prerequisites?

Reply:
This script was for the k8s test infrastructure nodes only and not meant to be invoked separately. We can separate it out further, as a test-cloud.sh, and prow-env.sh.

Reply:
I thought about splitting it up, but it didn't seem particularly useful as a separate run-cloud type of script - because a user can simply invoke the maven command to test against a cloud directly. This is only meant to be an entrypoint for prow, which relies on a specific pinned image which in turn requires this setup.

Reply:
The fact that it's a specific pinned image is very convincing. I drop this as a concern.